run.sh

```diff
@@ -1,3 +1,57 @@
 #!/bin/sh
+
 export PYTHONUNBUFFERED=1
+
+# As it appears parsing YAML with bash is not so easy task. The next function
+# parses simple YAML configurations files.
+#
+# Example YAML file (test.yml):
+# item_one: value1
+# item_two:
+#   sub_item: value3
+#
+# To parse the values you should use the next syntax:
+# eval $(yaml_parser test.yml "config_")
+#
+# Getting items:
+# echo $config_item_one
+#
+# To get nested items:
+#
+# echo $config_item_two_sub_item
+#
+# Note: There is "shyaml" python script to read YAML files, but we will probably
+# not use it in order not to have additional dependcies.
```

Comment:
It means that the client should install `shyaml` package in order to run `run.sh`.